src/clockcache.c

```diff
@@ -915,12 +920,20 @@ clockcache_assert_no_locks_held(clockcache *cc)
 void
 clockcache_assert_clean(clockcache *cc)
 {
-   uint64 i;
+#if SPLINTER_DEBUG
 
-   for (i = 0; i < cc->cfg->page_capacity; i++) {
-      debug_assert(clockcache_test_flag(cc, i, CC_FREE)
-                   || clockcache_test_flag(cc, i, CC_CLEAN));
+   for (uint64 i = 0; i < cc->cfg->page_capacity; i++) {
+
+      // We expect entry to be in only one of these two states.
+      entry_status entry_flag = clockcache_get_flag(cc, i);
+      debug_assert(((entry_flag & (CC_FREE | CC_CLEAN)) != 0),
```

Comment:
I had rewritten it so that the assert-message could print out the `entry_flag` status value.

As per your suggestion, reinstated the `debug_assert()` the way it was previously.

The `#if SPLINTER_DEBUG` is just to make sure that this code is only active in debug-builds. The rest of the code, i.e. body of `for()` is an empty block as all it contains is the `debug_assert()`.